mantidproject/fitbenchmarking
# Updated Readme

#### Description of Work

Fixes #128 
Provided explanations on how to solve this issue and made the repository main page nicer.

#### Testing Instructions

On a fresh mantid windows install, follow the instructions in the README.md on the repository main page and check if you can run fitbenchmarking.
